db/schema.rb

```diff
@@ -39,6 +39,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "orders", force: :cascade do |t|
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_orders_on_user_id"
+  end
```

Comment:
Brilliant implementation. Let's move on when everyone has the time to get back together on this.

I love you all.